### PR TITLE
Dataproc delay fix + add tests

### PR DESCRIPTION
**Pull request by VladaZakharova**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by lwyszomi — APPROVED** [review]

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py [hunk 1]
```diff
@@ -58,7 +61,17 @@
         region=REGION,
         batch=BATCH_CONFIG,
         batch_id=BATCH_ID,
-        timeout=5.0,
+    )
+    # [END how_to_cloud_dataproc_create_batch_operator]
+
+    # [START how_to_cloud_dataproc_create_batch_operator]
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py:67]: There should be only one marker with the name ``how_to_cloud_dataproc_create_batch_operator``. Either include all two operator examples in single START/END block or use different name (but then remember to also include it in the documentation).

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py [hunk 2]
```diff
@@ -68,6 +81,12 @@
     )
     # [END how_to_cloud_dataproc_get_batch_operator]
 
+    # [START how_to_cloud_dataproc_get_batch_operator]
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py:84]: +duplicated doc marker

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py [hunk 3]
```diff
@@ -83,7 +102,22 @@
     # [END how_to_cloud_dataproc_delete_batch_operator]
     delete_batch.trigger_rule = TriggerRule.ALL_DONE
 
-    create_batch >> get_batch >> list_batches >> delete_batch
+    # [START how_to_cloud_dataproc_delete_batch_operator]
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py:105]: duplicated doc marker - we don't need the second example of deleting batch in the docs, it looks trivial enough

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py (hunk 1 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py:67]: I thought if we have 2 examples then there should be doc strings for all of them :)
Thanks!
I have removed duplicated doc strings.

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py (hunk 2 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py]: Done

File: tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py (hunk 3 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py]: Done

**Pull request closed without merging**